### PR TITLE
Fixes/additions to redirects

### PR DESCRIPTION
**Pull request by bstuder99**
## Related issue #

Creates redirects for 404 links appearing at www.kubecost.com, and corrects some redirects which don't appear to be working.

### CODE REVIEW

File: .gitbook.yaml [hunk 1]
```diff
@@ -27,6 +27,10 @@ redirects:
     alerts.html: alerts.md
     assets: using-kubecost/navigating-the-kubecost-ui/assets.md
     troubleshoot-install: troubleshooting/troubleshoot-install.md
+    env: install-and-configure/install/environment.md
+    install-and-configure/install/custom-prom: install-and-configure/advanced-configuration/custom-prom.md
+    install-and-configure/install/custom-prom: install-and-configure/advanced-configuration/custom-prom/gcp-gmp-integration.md
```
Comment (davestrandburg) [inline .gitbook.yaml:32]: Should this one be:
install-and-configure/install/custom-prom**/gcp-gmp-integration**: install-and-configure/advanced-configuration/custom-prom/gcp-gmp-integration.md

**Review by davestrandburg — COMMENTED** [review]
LGTM - Just had one question on the custom-prom urls

**Comment by bstuder99** [comment]
That actually fixed an error I was having with GitBook sync, we should be good to merge now

**Review by davestrandburg — APPROVED** [review]

**Comment by davestrandburg** [comment]
Looks like you'll need Sean to approve to unblock

**Pull request merged**